database/factories/AirlineFactory.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Lightit\Backoffice\Airlines\Domain\Models\Airline;
+
+/** @extends Factory<Airline> */
+class AirlineFactory extends Factory
+{
+    protected $model = Airline::class;
+
+    public function definition()
+    {
+        return [
+            'name' => $this->faker->company,
+            'description' => $this->faker->company, 
+        ];
+    }
```

Comment:
Regarding what you did on `DatabaseSeeder.ts` a way to remove logic is to create a withCities function for example. And on the seeder you call

```
AirlineFactory::new()
            ->count(10)
            ->recycle($cities)
            ->withCities(4)
            ->create();
```            
So you can easily create Airlines with cities and prevent having extra logic on the `DatabaseSeeder` 

The `->recycle($cities)` will use the already created cities. 
Let me know if you don't understand and we can huddle it. 
```suggestion
    }
    
    public function withCities(int $count = 3): self
    {
        return $this->afterCreating(function (Airline $airline) use ($count) {
            $airline->cities()->attach(
                CityFactory::new()->count($count)->createMany()
            );
        });
    }
```